--- gecko_sdk/platform/Device/SiliconLabs/EFM32HG/Source/GCC/efm32hg.ld
+++ tomu.ld
@@ -29,8 +29,8 @@
 
 MEMORY
 {
-  FLASH (rx) : ORIGIN = 0x00000000, LENGTH = 65536
-  RAM (rwx)  : ORIGIN = 0x20000000, LENGTH = 8192
+  FLASH (rx) : ORIGIN = 0x00004000, LENGTH = 0xC000
+  RAM (rwx)  : ORIGIN = 0x20000000, LENGTH = 0x2000
 }
 
 /* Linker script to place sections and symbol values. Should be used together
